scripts/make-post.js

```diff
@@ -115,7 +115,8 @@ async function main() {
   const finalDate = new Date(datetime).toISOString();
 
   const postContent = `---
-title: "${title}"
+title: '${title}'
+slug: '${slug}'
```

Comment:
I was wondering where the current slugs come form, and realised it's the filename.
I think we can keep it as the source of truth.
Then I realised that Eleventy has already thought of that use case!

It should be available in the templates as `fileSlug`:
https://www.11ty.io/docs/data/#fileslug

Reply:
Thanks for the pointer. Helped me in figuring out that we can simply use `post.url` like you had in the past talks links.